centrifuge/sdk
# ci: pipeline bootstrap

### Description
CI/CD pipeline to:
- Build & Test
- Check naming convention and labels for the PR
- Create releases
- Publish to NPMwhen releases are validated

More details in: #10 

Extra:
- Added code coverage reports and uploads to CodeCov
